tests/contrib/operators/test_aws_athena_operator.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# 'License'); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+import unittest
+
+from airflow.contrib.operators.aws_athena_operator import AWSAthenaOperator
+from airflow import configuration
+
+try:
+    from unittest import mock
+except ImportError:
+    try:
+        import mock
+    except ImportError:
+        mock = None
+
+TEST_DETAILS = {
+    'task_id': 'test_aws_athena_operator',
+    'query': 'SELECT * FROM TEST_TABLE',
+    'database': 'TEST_DATABASE',
+    'region_name': 'ap-southeast-1',
+    'outputLocation': 's3://test_s3_bucket/',
+    'queue': 'athena_queue',
+    'pool': 'athena_pool'
+}
+
+
+class TestAWSAthenaOperator(unittest.TestCase):
+
+    def setUp(self):
+        configuration.load_test_config()
+
+        self.athena = AWSAthenaOperator(task_id='test_aws_athena_operator', query='SELECT * FROM TEST_TABLE',
+                                        database='TEST_DATABASE', region_name='ap-southeast-1',
+                                        output_location='s3://test_s3_bucket/', queue='athena_queue',
+                                        pool='athena_pool')
+
+    def test_init(self):
+        self.assertEqual(self.athena.task_id, TEST_DETAILS['task_id'])
+        self.assertEqual(self.athena.query, TEST_DETAILS['query'])
+        self.assertEqual(self.athena.database, TEST_DETAILS['database'])
+        self.assertEqual(self.athena.queue, TEST_DETAILS['queue'])
+        self.assertEqual(self.athena.pool, TEST_DETAILS['pool'])
+        self.assertEqual(self.athena.region_name, TEST_DETAILS['region_name'])
+        self.assertEqual(self.athena.aws_conn_id, None)
+
```

Comment:
We should probably use normal mocking to test the Hook, look at https://github.com/apache/incubator-airflow/blob/4fe1c61aea20cb4cd409a4db0fd47889fe27f795/tests/contrib/operators/test_sagemaker_training_operator.py#L110-L122 for an example